inst/builtin/bs5/shiny/_rules.scss

```diff
@@ -104,6 +108,10 @@ $bslib-checkbox-radio-margin-right: 0.35em !default;
   .bslib-card-box-shadow-none {
     --bslib-card-box-shadow: none;
   }
+
+  .popover {
+    box-shadow: var(--#{$prefix}box-shadow-sm);
```

Comment:
Maybe this should inherit from card shadow box?

Reply:
That's a good thought, but we want popovers to always use the small shadow box. Also the `--bslib-card-box-shadow` is unset at nested levels and we'd want to keep shadows on popovers in all cases.

Reply:
I agree with you about always using the small shadow box, @gadenbuie.  I did a little test to see what the other shadows would look like, and a nice thing is that it didn't bother me as much as I thought it would:

### Small:
![sm](https://github.com/rstudio/bslib/assets/5993637/0148b746-7a20-48e2-9da0-84cf568e777f)

### Medium:
![md](https://github.com/rstudio/bslib/assets/5993637/e0cc847e-a3fc-4be8-b15a-ff23eee95970)

### Large:
![lg](https://github.com/rstudio/bslib/assets/5993637/dfab297b-33e9-45df-af47-fcc9acc2b9e5)

Reply:
...Nice that it isn't an aesthetically breaking change if someone accidentally does that.

Reply:
I actually like the larger shadow for the popover, since popovers appear on top of all other UI elements, so they're sort of "closer" to the viewer than the rest of the app.

Reply:
Another popover shadow comparison in a dashboard context

### Large

![image](https://github.com/rstudio/bslib/assets/5420529/4d7498f9-f5cf-4aae-aea2-75e1ed1cd2c3)

### Medium

![image](https://github.com/rstudio/bslib/assets/5420529/c9aee930-9385-476f-a638-77df352bb3ab)

### Small

![image](https://github.com/rstudio/bslib/assets/5420529/ad681e95-8d33-4e35-ac20-5e365387b78c)